pkg/controller/cache.go

```diff
@@ -142,6 +142,24 @@ func (c *GVKRoutedCache) GetInformerForKind(ctx context.Context, gvk schema.Grou
 	return c.fallback.GetInformerForKind(ctx, gvk, opts...)
 }
 
+// RemoveInformer removes an informer entry and stops it if it was running.
```

Comment:
@sttts mind having a look at this as the original author of the rest of the code?

Reply:
Is this new method needed to satisfy an updated interface?

Reply:
Yes